dlib/CMakeLists.txt

```diff
@@ -385,7 +385,12 @@ if (NOT TARGET dlib)
                external/zlib/uncompr.c
                external/zlib/zutil.c
                )
-            if (NEON)
+
+            include(cmake_utils/use_arm_neon.cmake)
+            include(CheckCXXCompilerFlag)
+            CHECK_CXX_COMPILER_FLAG("-mfpu=neon" NEON_DEFINED)
```

Comment:
Yes. I mentioned it [before](https://sourceforge.net/p/dclib/discussion/442518/thread/b921bc27/#a9b0/ab19/64f9), but I overlooked it.